recaptcha_enterprise/demosite/pom.xml

```diff
@@ -43,6 +43,31 @@
     </dependency>
 
 
+    <!-- [Start_Selenium_dependencies] -->
```

Comment:
All region tags must use be prefixed with the product name `recaptchaenterprise`. Region tags format is also `[START region_tag]`

Reply:
Apologies for the confusion. Those are not region tags. Its just to mark the start and end of dependencies. I can modify the format to avoid confusion.